packages/core/src/modules/vc/W3cCredentialService.ts

```diff
@@ -352,6 +352,10 @@ export class W3cCredentialService {
     return result.map((record) => record.credential)
   }
 
+  public getKeyTypeByProofType(proofType: string): KeyType {
+    return this.suiteRegistry.getKeyTypeByProofType(proofType)
+  }
```

Comment:
a proof type could potentially have multiple supported key types associated with it. Should we make this an array?

Reply:
Good point, that will cause issues later on. Resolved now.